internal/api/describe.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"github.com/edulinq/autograder/internal/api/core"
+)
+
+// Routes must be validated before calling Describe.
```

Comment:
Nit: When referring to a function by their name, we usually end it with parens,
e.g. "Describe()".